### PR TITLE
Fix buckling crew

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

Cannot buckle people, bad

## Why It's Good For The Game

Fix bug

## Testing Photographs and Procedure

image

## Changelog
:cl:
fix: Can buckle people to stuff again
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Fix the can_buckle proc or remove it entirely

**Comment by itsmeow** [comment]
`can_buckle` is intended for use when ACTUALLY BUCKLING someone, not BEING buckled

**Comment by PowerfulBacon** [comment]
image

Can buckle checks if it can be buckled to something, the variable can_buckle indicates if something can be buckled to it (Which is very confusing).

**Comment by itsmeow** [comment]
Tested on new code, works

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1380,6 +1380,4 @@
 		to_chat(eminence, "<span class='brass'>You select [src].</span>")
 
 /mob/living/can_buckle()
-	if(IS_DEAD_OR_INCAP(src))
-		return FALSE
 	return TRUE
```
Comment (PowerfulBacon) [inline code/modules/mob/living/living.dm]: Just remove this

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**